chain-extensions/types/xvm/src/lib.rs

```diff
@@ -18,7 +18,7 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
-use astar_primitives::xvm::CallError;
+use astar_primitives::{xvm::CallError, Balance};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

LGTM!


<!-- This is an auto-generated comment by OSS CodeRabbit -->